package.json

```diff
@@ -13,10 +14,13 @@
     "precommit": "lint-staged",
     "toc": "doctoc README.md --github",
     "release": "rollout",
-    "build": "babel src --out-dir lib --delete-dir-on-start && npm run toc",
+    "build": "yarn build:commonjs && yarn build:modules && npm run toc",
+    "build:commonjs": "babel src --out-dir lib --delete-dir-on-start",
+    "build:modules": "BABEL_ENV=modules babel src --out-dir es --delete-dir-on-start",
```

Comment:
can we use `--envName esm` instead of the env var, it's a bit more cross platform

Reply:
Good catch! Done.